### PR TITLE
Add cmc-citizen-frontend to prod build monitor

### PR DESCRIPTION
**Pull request by markry-kainos**
Adding `cmc-citizen-frontend` to buildMonitor in preparation for nightly jenkins pipeline.

### CODE REVIEW

File: cac-prod.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jenkins:
         name: all
     - buildMonitor:
         includeRegex: >-
-          ^HMCTS_.*(Platform|CNP).*\/(draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template|cnp-rhubarb-recipes-service)\/master
+          ^HMCTS_.*(Platform|CNP).*\/(cmc-citizen-frontend|draft-store|cmc-pdf-service|feature-toggle-api|private-beta-invitation-service|service-auth-provider-app|spring-boot-template|cnp-rhubarb-recipes-service)\/master
```
Comment (timja) [inline cac-prod.yml]: this is the platform team build monitor, you want your own one

Comment (markry-kainos) [reply cac-prod.yml]: oops - blindly adding

File: cac-prod.yml [hunk 2]
```diff
@@ -129,6 +129,12 @@ jenkins:
         name: ETHOS
         recurse: true
         title: ETHOS
+    - buildMonitor:
+        includeRegex: >-
+          ^HMCTS_.*(CMC).*\/(cmc-citizen-frontend)\/master
```
Comment (timja) [inline cac-prod.yml]: why only citizen frontend?

Comment (markry-kainos) [reply cac-prod.yml]: good point - we will be adding to all actually

Comment (markry-kainos) [reply cac-prod.yml]: ill add others after we have citizen working as we want actually 👍

**Comment by markry-kainos** [comment]
@timja pls merge :)

**Pull request merged**